.github/workflows/validate.yml

```
@@ -22,16 +22,29 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
         
+      - name: Install validation tools
+        run: npm install -g ts-prune depcheck
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)